src/platform/packages/shared/kbn-unified-tabs/src/constants.ts

```diff
@@ -10,6 +10,6 @@
 export const MAX_TAB_LABEL_LENGTH = 500;
 
 export const MAX_TAB_WIDTH = 280;
-export const MIN_TAB_WIDTH = 96;
+export const MIN_TAB_WIDTH = 112;
```

Comment:
To follow up on the Slack conversation, one option if we wanted to use actual EUI vars would be to change the consts to something like a multiplier of the base value, then pass the base value to the utils that use them from the consuming React components. Something along these lines:
```tsx
// constants.ts
export const MIN_TAB_WIDTH_MULTIPLIER = 7;

// component
const minTabWidth = MIN_TAB_MULTIPLIER * euiTheme.base;
```

That said, it's a bit convoluted and I don't feel strongly about it, so fine to keep it as is too.

Reply:
Most of those places would need to multiply it within a component, right? And the `calculate_responsivene_tab.ts` would face the same problem as constants here
<img width="375" alt="Screenshot 2025-06-12 at 13 32 25" src="https://github.com/user-attachments/assets/9d7e723c-88fe-45b6-92c2-dff47c6c2ba3" />

Reply:
Leaving the number then, thanks for brainstorming together!

Reply:
Sounds good!